src/calibre/gui2/store/stores/archive_org_plugin.py

```diff
@@ -1,6 +1,5 @@
 # -*- coding: utf-8 -*-
 
-from __future__ import (unicode_literals, division, absolute_import, print_function)
```

Comment:
Why have the __future__ imports been removed from various store plugins?

Reply:
Because my sed misfired, apparently. :/